addons/seashell.json

```diff
@@ -0,0 +1,30 @@
+{
+  "id": "seashell",
+  "name": "Seashell",
+  "description": "An easy and quick way to run shell commands. For advanced users. ",
+  "author": "Flatsiedatsie",
+  "homepage_url": "https://github.com/flatsiedatsie/seashell",
+  "license_url": "https://github.com/flatsiedatsie/seashell/blob/master/LICENSE",
+  "primary_type": "extension",
+  "packages": [
+    {
+      "architecture": "linux-arm",
```

Comment:
Is there a reason it's `linux-arm` only? This could really be applicable to any platform. The only platform-specific bit is the restart function.

Reply:
Convenience, since I develop for the Raspberry Pi :-)

Reply:
Maybe in the future, the restart functionality could just be disabled on other platforms.